### DISCUSSION TITLE
Running Laravel Reverb

### DISCUSSION THREAD

**Discussion by connor-devuk** [coollabsio/coolify#2717] [Q&A]
Hey guys, I'm not too sure whats the best way of running something ilke Laravel Reverb with my application on coolify any pointers in terms of setting something like this up?

**Comment by occult** [comment]
This post helped me out 

https://blog.gaetanlittiere.com/posts/reverb-in-production-with-coolify/

**Reply by KwaminaWhyte** [reply]
Page doesn't exist

**Reply by occult** [reply]
@KwaminaWhyte add this to your `nixpacks.toml`

```
[program:reverb]
process_name = %(program_name)s_%(process_num)02d
command = bash -c 'exec php /app/artisan reverb:start'
autostart = true
autorestart = true
stopasgroup = true
killasgroup = true
numprocs = 1
startsecs = 0
stopwaitsecs = 3600
stdout_logfile =/var/log/reverb.log
stderr_logfile =/var/log/reverb.log
'''
```

**Reply by KwaminaWhyte** [reply]
Alright, thanks

**Reply by tiagomichaelsousa** [reply]
I managed to get reverb running with supervisor and nixpacl configuration, but I’m still unsure about the remaining configurations. For example, which ports need to be open on the server, how to handle SSL certificates, or whether we should expose any ports in the Docker Compose file or Coolify configuration. Do you guys have a tutorial or guide on this? I couldn’t find anything so far. Once I get everything configured, I’d be happy to write a blog post or even contribute to the Coolify documentation to make this process clearer for everyone!

**Reply by Gmk0** [reply]
Hello, have you found a way to expose a port or how to use it

**Reply by tiagomichaelsousa** [reply]
Hello there! I managed to get it running, but I'm not entirely sure if it's the best approach. I deployed a new container of the application under a subdomain and updated the environment variables. To check if your reverb server is functioning, deploy the application and visit the URL where you deployed it. You should see a **Not found.** response.

From there, it's just a matter of tweaking your environment variables and ensuring that your frontend is correctly set up and receiving the events in the network tab of your console through the socket filter.

**Reply by Gmk0** [reply]
Have you used Nixpacks?

**Reply by tiagomichaelsousa** [reply]
No, i'm using a docker-compose build but i also want to change that because it doesn't support rolling update and it creates some downtime when deploying

**Reply by Sy-Kazmi** [reply]
> This post helped me out
> 
> https://blog.gaetanlittiere.com/posts/reverb-in-production-with-coolify/

This link is not working it says you dont have permission

**Reply by Gmk0** [reply]
I was able to find an alternative
